plugins/ruby/CMakeLists.txt

```diff
@@ -3,9 +3,9 @@ IF (APPLE)
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
     SET(RUBYLIB ${CMAKE_CURRENT_BINARY_DIR}/libruby.dylib)
 ELSEIF(UNIX)
-    EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E tar xzf ${CMAKE_CURRENT_SOURCE_DIR}/libruby187.linux.tar.gz
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
-    FILE(RENAME ${CMAKE_CURRENT_BINARY_DIR}/libruby1.8.so.1.8.7 ${CMAKE_CURRENT_BINARY_DIR}/libruby.so)
+    #EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E tar xzf ${CMAKE_CURRENT_SOURCE_DIR}/libruby187.linux.tar.gz
+    #    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+    #FILE(RENAME ${CMAKE_CURRENT_BINARY_DIR}/libruby1.8.so.1.8.7 ${CMAKE_CURRENT_BINARY_DIR}/libruby.so)
```

Comment:
This PR must be based on a really old branch, because "tar" doesn't appear anywhere in plugins/ruby/CMakeLists.txt. I'll look at the stockpiles changes in a bit.

Reply:
I have no memory at all of why I did this, or why the pull request exists. Sorry!